src/SpacetimeDBClient.cs

```diff
@@ -954,4 +1086,22 @@ public void FrameTick()
             }
         }
     }
+
+    internal struct UintAllocator
+    {
+        /// <summary>
+        /// Interlocked.Increment only works with ints.
+        /// </summary>
+        private int lastAllocated;
+
+        /// <summary>
+        /// Allocate a new ID in a thread-safe way.
+        /// </summary>
+        /// <returns>A previously-unused ID.</returns>
+        public uint Next()
+        {
+            var next = Interlocked.Increment(ref lastAllocated);
```

Comment:
Huh, that's a good one. I guess StartTrackingRequest should be doing the same, and might be potentially racy already (although it's less critical functionality).

Maybe worth switching its `_nextRequestId` to your `UintAllocator` as well.